server/controllers/ArticleController.js

```diff
@@ -272,5 +274,115 @@ class ArticleController {
         Error: err.toString()
       }));
   }
+
+  /**
+   *
+   *
+   * @static
+   * @param { object } req
+   * @param { object } res
+   * @description add article to user favorite list
+   * @memberof ArticleFixture
+   * @returns { object } object
+   */
+  static addFavourite(req, res) {
+    const articleId = req.params.id;
+    const userId = req.decoded.id;
+    console.log(userId);
+    Article.findOne({
+      where: { id: articleId }
+    })
+      .then(article => article)
+      .then((article) => {
+        if (!article) {
+          return res.status(404).json({ status: 'Success', message: 'Article does not exist' });
+        }
+        return FavoriteArticle.findOrCreate({
+          include: [{
+            model: Article,
+          }],
+          where: { userId, articleId }
+        });
+      })
+      .then((result) => {
+        res.status(200)
+          .json({
+            status: 'Success',
+            message: 'Article added to favorite',
+            article: result,
+          });
+      })
+      .catch(() => res.status(500)
+        .json(
+          {
+            status: 'Failed',
+            message: 'Problem favouriting article',
+          }
+        ));
+  }
+
+  /**
+   *
+   *
+   * @static
+   * @param { object } req
+   * @param { object } res
+   * @description remove an article from user list
+   * @memberof ArticleFixture
+   * @returns { object } object
+   */
+  static removeFavourite(req, res) {
+    const userId = req.decoded.id;
+    const articleId = req.params.id;
+    Article.findOne({
+      where: { id: articleId }
+    })
+      .then(article => article)
+      .then((article) => {
+        if (!article) {
+          return res.status(404).json({ status: 'Success', message: 'Article does not exist' });
+        }
+        return FavoriteArticle.destroy({
+          where: { userId, articleId }
+        });
+      })
+      .then(() => res.status(200)
+        .json(
+          {
+            status: 'Success',
+            message: 'Article removed from favourite',
+          }
+        ))
+      .catch(() => res.status(500)
+        .json(
+          {
+            status: 'Failed',
+            message: 'Problem removing article from favorite'
+          }
+        ));
+  }
+
+  /**
+   *
+   *
+   * @static
+   * @param { object } req
+   * @param { object } res
+   * @description returns all user favorite articles
+   * @memberof ArticleFixture
+   * @returns { object } object
+   */
+  static getAllFavorite(req, res) {
+    const userId = req.decoded.id;
+    FavoriteArticle.findAll({
+      include: [
+        {
+          model: Article,
+        },
+      ],
+      where: { userId, }
+    }).then(result => res.status(200).json({ status: 'Success', result }))
```

Comment:
#### Can we not make this a one-liner please?
This should read:
```js
.then(result => {
  res.status(200)
      .json({
        status: 'Success',
        result
      })
})
```
This also apply to the next line of code and every where in the code base that this was implemented.

Reply:
@Victor-Ugwueze 
You did not fix this??!!!